### PR TITLE
Docker & WordPress – Containerizing WordPress with Docker-Compose

### PR DESCRIPTION
**Pull request by Rose-stack**
### Article checklist
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by Rose-stack** [comment]
This fixes #1757

**Comment by Rose-stack** [comment]
I am still waiting for the reviews. How long does it take, or how is the process done. I see some latest articles are already reviewed, and this one has a couple of days with no checks.

**Comment by WanjaMIKE** [comment]
It needs to be checked for plagiarism first @Rose-stack

File: articles/docker-wordpress-containerizing-wordpress-with-docker-compose/index.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
+Docker is a container management system. It is used to manage an application that has multiple components. Docker-compose is a toolkit provided by Docker. It defines and runs a multi-container Docker application. Most applications involve more than one architectural component. Take a simple, hardy project that needs a database and a back-end server of some kind.
```
Comment (WanjaMIKE) [inline articles/docker-wordpress-containerizing-wordpress-with-docker-compose/index.md:1]: Could you add a link to the Docker site so as the user can learn more from it.

**Comment by Rose-stack** [comment]
I have added the Docker Documentation link @WanjaMIKE

**Review by WanjaMIKE — APPROVED** [review]
Nice article Rose. I have learned a lot about containerization using Docker compose.

**Comment by hectorkambow** [comment]
plagiarism check done

**Pull request merged**